doc/odoc_for_authors.mld

```diff
@@ -773,12 +775,12 @@ Note: A comment attached to a declaration shouldn't contain any heading.
 
 {2:synopsis Synopsis}
 
-The synopsis of a module (a module type, a class, etc..) is the first
+A module's synopsis (a module type, a class, etc..) is the first
```

Comment:
The part in parentheses are alternatives to "module". So I suggest dropping it and adding a sentence saying "This also applies to module types, classes, etc..".